cli/src/clojure_lsp/server.clj

```diff
@@ -343,8 +338,7 @@
                                         (.setSignatureHelpProvider (SignatureHelpOptions. []))
                                         (.setCallHierarchyProvider true)
                                         (.setLinkedEditingRangeProvider true)
-                                        (.setCodeActionProvider (doto (CodeActionOptions. (vec (vals coercer/code-action-kind)))
-                                                                  (.setResolveProvider true)))
```

Comment:
The server no longer advertises that it can resolve code actions, which I hope will ensure editors don't try that path 🤞. Of course, this could be turned back on someday, if needed.

Reply:
I think we still need to keep the CodeActionOptions

Reply:
exactly